composer.json

```diff
@@ -16,7 +16,8 @@
         }
     ],
     "require": {
-        "php": ">=5.6"
+        "php": "^7.2",
+        "laravel/framework": "~5.4.0|~5.8.0|~5.9.0|^6.0"
```

Comment:
Please remove the php requirement, and change laravel/framework to ^5.4|^6.0.

Thanks :)

Reply:
Ok sure.

Reply:
I am setting the php as default. 
"php": ">=5.6"